src/sonic-ctrmgrd/ctrmgr/container

```diff
@@ -0,0 +1,367 @@
+#!/usr/bin/python3 -u
+
+import argparse
+import docker
+import inspect
+import json
+import os
+import sys
+import syslog
+import time
+
+from swsscommon import swsscommon
+
+sys.path.append('/usr/share/sonic/scripts/')
+import container_state
+
+state_db = None
+
+# DB field names
+FEATURE_TABLE = "FEATURE"
+SET_OWNER = "set_owner"
+NO_FALLBACK = "no_fallback_to_local"
+
+CURRENT_OWNER = "current_owner"
+UPD_TIMESTAMP = "update_time"
+DOCKER_ID = "container_id"
+REMOTE_STATE = "remote_state"
```

Comment:
can you remind me why is this named as remote state?

Reply:
remote_state is only used by instances deployed by remote-ctr-management, which in our case "kubernetes", hence named so.
none/pending/running/stopped.
As for local, start/stop being synchronous, there are only two states (binary) which is reflected by presence/absence of container-id.